tools/cldr-code/src/test/java/org/unicode/cldr/util/TestPathDescription.java

```diff
@@ -1,13 +1,34 @@
 package org.unicode.cldr.util;
 
+import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
 
 public class TestPathDescription {
     @Test
     public void testParseGiantString() {
-        RegexLookup<String> l = PathDescription.parseLookupString();
-        assertNotNull(l);
+        final PathDescriptionParser parser = new PathDescriptionParser();
+        RegexLookup<Pair<String, String>> lookup =
+                parser.parse(PathDescription.getPathDescriptionString());
+        assertNotNull(lookup);
+        assertFalse(lookup.size() == 0, "lookup is empty");
+        String references = parser.getReferences();
+        assertNotNull(references);
+        assertFalse(parser.getReferences().trim().isBlank());
```

Comment:
Very minor: .trim().isBlank() is redundant, I think.

Reply:
BTW: we should check all of our .md files for URLs containing "#h.", and raise errors when found. Do you want me to file a separate tickets?

Reply:
yes would be separate -- could just be a search though?